ray-operator/main.go

```diff
@@ -134,13 +137,21 @@ func main() {
 
 		pe := zap.NewProductionEncoderConfig()
 		pe.EncodeTime = zapcore.ISO8601TimeEncoder
-		consoleEncoder := zapcore.NewConsoleEncoder(pe)
+		var fileEncoder zapcore.Encoder
+		if logFileEncoder == "json" {
+			fileEncoder = zapcore.NewJSONEncoder(pe)
+		} else if logFileEncoder == "console" {
+			fileEncoder = zapcore.NewConsoleEncoder(pe)
+		} else {
+			setupLog.Info("invalid encoder for log file")
```

Comment:
Should we specify what the valid values are?

Reply:
added